### ISSUE TITLE
Update room request tags parameter format.

### ISSUE THREAD

**Issue by amatkivskiy** [gitterHQ/docs#31]
Hi guys,

I think it will be cool to change **tags** parameter format in [update room](https://github.com/gitterHQ/docs/blob/master/03.Rooms-resource.md#update-room) request from one string with values separated by come to simple array with string tags.

Current:
```json
{
  "tags": "tag1, tag2, tag3"
}
```
Suggested: 
```json
{
  "tags": [
    "tag1",
    "tag2",
    "tag3"
  ]
}
```

What do you think about this?

**Comment by Odonno** [comment]
@amatkivskiy Yes, it will be more consistent. I didn't know why the update doesn't work until I found your issue.
